themis_lock_manager.go

```diff
@@ -93,7 +93,8 @@ func (m *themisLockManager) GetCommitTimestamp(cc *hbase.ColumnCoordinate, prewr
 	// add del write column
 	g.AddStringColumn("#d", qual)
 	// time range => [ours startTs, +Inf)
-	g.AddTimeRange(prewriteTs, math.MaxInt64)
+	g.AddTimeRange(prewriteTs, math.MaxInt32)
```

Comment:
Java Hbase client use long as timerange parameter:
https://hbase.apache.org/apidocs/org/apache/hadoop/hbase/client/Get.html#setTimeRange(long,%20long)

long is a 64bits integer: https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html

Why change this ftom math.MaxInt64 to MaxInt32?